<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MV_TestClass</class>
 <widget class="QMainWindow" name="MV_TestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MV_Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="inputheaderlable">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Input Fields</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="displayheaderlable">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Display Fields</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="radioButton_A">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Option A</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Option C</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_B_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Option B</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_D">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Option D</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="radiobuttonLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Option A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Option 1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Option 2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Option 3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Option 4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="checkboxLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Option 1</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>241</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addItemButton">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>410</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Item</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>291</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Surname</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addItem_Table">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Item</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuOption_1">
     <property name="title">
      <string>Option 1</string>
     </property>
     <addaction name="action1_1"/>
     <addaction name="action1_2"/>
     <addaction name="action1_3"/>
    </widget>
    <addaction name="menuOption_1"/>
    <addaction name="Menu_Option2"/>
    <addaction name="separator"/>
    <addaction name="Menu_Beenden"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Menu_Option2">
   <property name="text">
    <string>Option 2</string>
   </property>
  </action>
  <action name="Menu_Beenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="action1_1">
   <property name="text">
    <string>1.1</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>1.2</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>1.3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MV_Test.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
